### PR TITLE
:book: Document expectations around PRs

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates the CONTRIBUTING.md doc to explicitly state some expectations regarding PRs.

## Related issue(s)

Fixes #

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,6 +12,29 @@ Origin (DCO). This document was created by the Linux Kernel community and is a
...
+author(s) before merging.  Typically this means an LGTM from someone
+else who does a comprehensive review.  The author(s) should not LGTM
+their own work.
+
+A PR should get prompt review.  That includes an initial reaction
+within one business day.
+
 ## Prerequisites
 
 [Install Go](https://golang.org/doc/install) 1.19+.
```
Comment (ezrasilvera) [inline CONTRIBUTING.md:40]: Is this work-in progress?  The Prerequisites  section seems a little unrelated

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,6 +12,29 @@ Origin (DCO). This document was created by the Linux Kernel community and is a
...
 
+## Expectations around PRs
+
+A branch submitted in a PR should consist only of a strict chain of
+commits built on a commit that was in the `main` branch --- no merges
+into the chain.  Rebase if necessary.  Do not make the chain
+unnecessarily long; squash together commits that have no useful
+distinction going forward.
+
+Unlike the automation for Kubernetes, our CI automation does not
```
Comment (ezrasilvera) [inline CONTRIBUTING.md:23]: I suggest to move this section to the end. It is related only to people in OWNERS and is not relevant to most people. I think the next section about the LGTM requirements should come first.
Maybe even put it with some prefix of "Note to OWNERS ...."  or something

**Comment by MikeSpreitzer** [comment]
The CONTRIBUTING.md file has been updated to address this issue.

**Pull request closed without merging**